challenge_7/java/zmiller91/MissingNumber.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * @author zmiller
+ */
+public class MissingNumber {
+    
+    public static int find(int[] input) {
+        
+        // Invalid
+        if (input.length == 0) {
+            return -1;
+        }
+        
+        // Find the sum, min, and max values
+        int max = Integer.MIN_VALUE;
+        int actualSum = 0;
+        for(int i : input) {
+            actualSum += i;
+            max = Math.max(max, i);
```

Comment:
How would I know that 4 is missing? The directions were a bit unclear. To me, it appears that sequence is correct. To be more clear, I think you should say the `N` is defined as the length of the list. Leaving `N` ambiguous makes me assume it can be anything from 0 to N where N is the largest number in the list.

I'll correct the solution, though.